tfm/repro/TestRepro.cxx

```diff
@@ -48,74 +48,14 @@ TfmProxyConfig::TfmProxyConfig(AbstractDb* db, const CommandLineParser& args)
    insertConfigValue("QValueMsBetweenForkGroups", "5000");
    insertConfigValue("QValueMsBeforeCancel", "5000");
 
+   insertConfigValue("TimerC", "30");
```

Comment:
I wonder if we should put this longer than the 32 second transaction timeout, just to make sure transaction timeouts have a chance to occur first.  ie:  Set to 40 instead?

Reply:
Good point, will test it out with the value of `40`.

Reply:
Increased TimerC value to 40